edenai_apis/apis/anthropic/anthropic_api.py

```diff
@@ -239,3 +326,71 @@ def text__chat(
                 original_response=None,
                 standardized_response=StreamChat(stream=stream_response),
             )
+
+    def multimodal__chat(
+        self,
+        messages: List[ChatMultimodalMessageDataClass],
+        chatbot_global_action: Optional[str],
+        temperature: float = 0,
+        max_tokens: int = 25,
+        model: Optional[str] = None,
+        stop_sequences: Optional[List[str]] = None,
+        top_k: Optional[int] = None,
+        top_p: Optional[int] = None,
+        stream: bool = False,
+        provider_params: Optional[dict] = None,
+    ) -> ResponseType[Union[ChatMultimodalDataClass, StreamChatMultimodal]]:
+
+        formated_messages = self.__format_anthropic_messages(messages=messages)
+        body = {
+            "anthropic_version": "bedrock-2023-05-31",
```

Comment:
it would be nice to fetch the version from constraints, instead of hardcoding here

Reply:
should be done